ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -38,20 +33,22 @@
 
   private final Config config;
   private final RunnerBuilder runnerBuilder;
-
-  public EthSigner(final Config config, final RunnerBuilder runnerBuilder) {
+  private final TransactionSigner signer;
+  private final Vertx vertx;
+
+  public EthSigner(
+      final Config config,
+      final TransactionSigner signer,
+      Vertx vertx,
```

Comment:
nit: final on vertx arg

Reply:
I have to get used to the final thing!!!!
done!